spec/person_spec.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'spec_helper'
+
+describe Person do
+  before :each do
+    @person = Person.new(24)
+  end
+
```

Comment:
- [ ] There are about 3 or 4 class methods in our person class which needs to be tested. Kindly write test for that :100: 

![Screenshot from 2022-09-14 14-03-17](https://user-images.githubusercontent.com/46388113/190175767-2cf693cf-82cd-4f87-8d9c-eccc25042110.png)
For more clarity.....